ibc/main_test.go

```diff
@@ -73,6 +91,17 @@ func prepareEnvironment(
 	err = testPersistenceMod.Start()
 	require.NoError(t, err)
 
+	testConsensusMod := newTestConsensusModule(bus)
+	err = testConsensusMod.Start()
+	require.NoError(t, err)
+
+	// TODO: I want to mock this as using a real p2p module is hard to test
+	/**
+	testP2PMod := newTestP2PModule(bus)
+	err = testP2PMod.Start()
+	require.NoError(t, err)
+	**/
```

Comment:
nit: We should prefer not to commit commented code because it's much more difficult to determine when it becomes obsolete.

I realize this PR is still marked as a draft but I'm reviewing the same, since you explicitly asked for review. :sweat_smile:

Reply:
Yeah I had to commit it as me and @Olshansk did some work on the branch together the other day to get the P2P mock working in the test. Unsure whether I should rebase a WIP or not after the other day o-o